README.md

```diff
@@ -25,4 +25,103 @@ First of all, you need to require this library through Composer:
 composer require nexylan/gandi-sdk
 ```
 
-After this, you can use it as is.
+With Symfony:
+
+Enable the bundle on the `AppKernel` class:
+
+```php
+// app/AppKernel.php
+
+public function registerBundles()
+{
+    $bundles = array(
+        // ...
+        new Nexy\SlackBundle\NexySlackBundle(),
+    );
+
+    // ...
+
+    return $bundles
+}
+```
+
+## Configuration
+
+Configure the bundle to your needs:
+
+```yaml
+# parameters.yml
+parameters:
+    # Change to https://rpc.gandi.net/xmlrpc/ in prod
+    gandi_rpc_url: https://rpc.ote.gandi.net/xmlrpc/
+```
+
+
+```yaml
+# config.yml
+nexy_gandi:
+    api_url: %gandi_rpc_url%
+    api_key: 'yourApiKey'
+```
+
+## Usage
+
+Use the predefined methods and/or use Gandi methods directly
+
+```php
+$gandi = new Gandi('api_url', 'api_key');
+
+$gandi->setup()->domain->info('mydomain.net');
```

Comment:
`$result = $gandi...`

And the result extract is not necessary. But If you want to do it, it should be commented.

Maybe you can show a smaller one.